test/wargames/ropsten.test.ts

```diff
@@ -389,6 +404,82 @@ describe("ropsten deployment", function () {
+      const eye = await tokenFactory.attach(fetchAddress("EYE")) as Types.MockToken
+
+      const eyeBalance = await eye.balanceOf(owner.address)
+      logger("eye balance", eyeBalance)
+
+      const flanFactory = await ethers.getContractFactory("Flan")
+      const flan = await flanFactory.attach(fetchAddress("Flan")) as Types.Flan
+      const tokenProxyRegistryFactory = await ethers.getContractFactory("TokenProxyRegistry")
+      const tokenProxyRegistry = await tokenProxyRegistryFactory.attach(fetchAddress("TokenProxyRegistry")) as Types.TokenProxyRegistry
+      const set = await tokenProxyRegistry.tokenProxy(flan.address)
+      logger('set', `${flan.address}, ${set.behodlerProxy}`)
+      await flan.approve(set.behodlerProxy, ethers.constants.MaxUint256)
+
+      const flanBalance = await flan.balanceOf(owner.address)
+      logger('owner flan balance', flanBalance)
+      const SwapComparisonsFactory = await ethers.getContractFactory("SwapComparisons")
+      const swapComparisons = await deploy<Types.SwapComparisons>(SwapComparisonsFactory, fetchAddress("TokenProxyRegistry"), fetchAddress("EYE"), fetchAddress("DAI"), fetchAddress("Behodler"), fetchAddress("Flan"))
+
+      await eye.approve(swapComparisons.address, ethers.constants.MaxUint256)
+      await swapComparisons.swapMeasureSimpleSwap()
+      const simpleGasConsumption = await swapComparisons.gasConsumed(true)
+      logger('simple gas consumed', simpleGasConsumption.toString())
+
+      await flan.approve(swapComparisons.address, ethers.constants.MaxUint256)
+      await swapComparisons.swapMeasureCliffFaceSwap()
+      const cliffFaceGasConsumed = await swapComparisons.gasConsumed(false)
+      logger('cliffFace gas consumed ', cliffFaceGasConsumed.toString())
+      const percentage = simpleGasConsumption.mul(100).div(cliffFaceGasConsumed)
+      logger('cliffFace gas consumed ', cliffFaceGasConsumed.toString())
+
+      logger(`simple uses ${percentage.toString()}% as much gas as cliffFace. Note different base tokens`)
+    })
+
+  })
+  interface ethersLib {
+    [key: string]: string
+  }
+
+  const getAmountOut = (
+    amountIn: BigNumber,
+    reserveIn: BigNumber,
+    reserveOut: BigNumber
+  ): BigNumber => {
+
+    const amountInWithFee: BigNumber = amountIn.mul(995);
+    const numerator = amountInWithFee.mul(reserveOut);
+    const denominator = (reserveIn.mul(1000)).add(amountInWithFee);
+    const amountOut = numerator.div(denominator);
+    return amountOut
+  }
+
+  const getBehodler = async (getContract: <T extends Contract> (contractName: contractNames, factoryName?: string, libraries?: ethersLib) => Promise<T>): Promise<Types.Behodler> => {
+    const addressBalanceCheck = await getContract("AddressBalanceCheck")
+    const behodlerLib: ethersLib = {
+      AddressBalanceCheck: addressBalanceCheck.address
+    }
+    return await getContract("Behodler", "Behodler", behodlerLib) as Types.Behodler
+  }
```

Comment:
Both `getAmountOut` and `getBehodler` functions don't seem to be used anywhere. Why do we need them?

Reply:
removed unused functions